tracer/src/Datadog.Trace/Iast/IastTags.cs

```diff
@@ -19,6 +19,9 @@ internal partial class IastTags : CommonTags
     [Tag(Tags.IastJsonTagSizeExceeded)]
     public string? IastJsonTagSizeExceeded { get; set; }
 
+    [Tag(Tags.IastMetaStructTagSizeExceeded)]
+    public string? IastMetaStructTagSizeExceeded { get; set; }
```

Comment:
Adding fields to the `TagsList` types is an optimization that speeds up access at the cost of allocating more bytes on the heap (the type becomes larger with every field). I think this should only be used for very common tags (tags that are always or almost always added to the span). Adding tags that are the exception instead of the rule will incur the additional allocated bytes bytes for very little benefit.

In other words, if you don't except to use the `IastMetaStructTagSizeExceeded` often, you should consider not adding the field here. Same for `IastJsonTagSizeExceeded` above.